angelsbioprocessing/prototypes/items/bio-refugium.lua

```diff
@@ -4,7 +4,7 @@ data:extend(
   -- PROCESSING ---------------------------------------------------------------
   -----------------------------------------------------------------------------
   {
-    type = "item",
+    type = "capsule",
```

Comment:
not 100% sure this is wanted, but the capsule properties below this indicate that this is a healing capsule, and not an "item" per se.

Reply:
it was a capsule before, but was later changed as people found it hard to insert meat into buildings (eating it instead)

Reply:
I see, in that case, I'm happy to remove the capsule properties, using fish should be fine for healing anyway.